source/common/upstream/cds_api_impl.cc

```diff
@@ -1,65 +1,53 @@
 #include "common/upstream/cds_api_impl.h"
 
-#include <chrono>
 #include <string>
-#include <vector>
 
-#include "common/common/assert.h"
-#include "common/config/cds_json.h"
+#include "common/config/subscription_factory.h"
 #include "common/config/utility.h"
-#include "common/http/headers.h"
-#include "common/json/config_schemas.h"
-#include "common/json/json_loader.h"
+#include "common/upstream/cds_subscription.h"
 
 namespace Envoy {
 namespace Upstream {
 
 CdsApiPtr CdsApiImpl::create(const Json::Object& config, const Optional<SdsConfig>& sds_config,
                              ClusterManager& cm, Event::Dispatcher& dispatcher,
                              Runtime::RandomGenerator& random,
-                             const LocalInfo::LocalInfo& local_info, Stats::Scope& scope) {
+                             const LocalInfo::LocalInfo& local_info, Stats::Store& store) {
```

Comment:
I needed to be able to generate a scope with the `cluster_managed.cds` prefix to pass to `Config:Utility::generateStats` below.